### DISCUSSION TITLE
Heliocentric ecliptic longitudes

### DISCUSSION THREAD

**Discussion by NablaDeltaPsi** [Stellarium/stellarium#2201] [Ideas]
The currently displayed [**ecliptic longitude**s](https://en.wikipedia.org/wiki/Ecliptic_coordinate_system#Spherical_coordinates) are **_geocentric_ ecliptic longitudes**. The **_heliocentric_ ecliptic longitudes** objectively describe the position of planets in the solar system and are IMO worth being added for each planet.

I'm new to github, how can I request such a feature to be added by someone?

**Comment by alex-w** [comment]
If someone want to know heliocentric ecliptic coordinates of planets, then he can move on the Sun and check coordinates. Want to see areocentric ecliptic coordinates? No problem, just move on Mars and check coordinates.

**Reply by NablaDeltaPsi** [reply]
Nice! Thanks, didn't know I can move to the sun. Is it also possible to move to the _center_ of the sun? Setting height to -696.350.000 meters? :D

**Comment by gzotti** [comment]
Actually the longitudes are topocentric if you have not explicitly switched off topocentric correction. 

I see no immediate application for human observers on earth for these data, but they are probably available via scripting, or yes, observe from the Sun. When you are there, switch off topocentric correction and you will be in the Solar centre.
